docs/requirements.txt

```diff
@@ -1,2 +1,3 @@
 Sphinx==5.1.0
 sphinx-rtd-theme==1.0.0
+web-poet
```

Comment:
why is it required? I'm worried if it'd break something, insyalling we-poet from pypi instead of doing it from source.

Reply:
It's for importing `web_poet.page_inputs.url._Url` to ensure that the string converter only handles those.

Reply:
Ah, I see. Shouldn't we add it to install_requires instead? It seems otherwise utils.py will fail with ImportError

Reply:
I was confused; for some reason I was thinking this is a PR to web-poet 🤦

Reply:
Good call! moved it to `install_requires`. https://github.com/zytedata/zyte-common-items/pull/11/commits/a23757e2080ec986c707bbe482c6268d23e1aa94